Hologram/Network/Modem/Modem.py

```diff
@@ -57,6 +57,7 @@ class Modem(IModem):
         0x2F: u'\\',
     }
 
+    # We use device_name instead of serial port, if device name is set we skip searching for a serial port
```

Comment:
This comment is a confusing to me; what is the difference between initializing the class with/without a device_name and how would I decide if I need to pass a device_name or not?